microk8s-resources/actions/enable.reg.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -e
+
+source $SNAP/actions/common/utils.sh
+
+read -ra ARGUMENTS <<< "$1"
+argz=("${ARGUMENTS[@]/#/--}")
+
+# check if reg cli is already in the system.  Download if it doesn't exist.
+if [ ! -f "${SNAP_DATA}/bin/reg" ]; then
+  REG_VERSION="${REG_VERSION:-v0.16.0}"
+  echo "Fetching reg version $REG_VERSION."
+  sudo mkdir -p "$SNAP_DATA/bin"
+  sudo "${SNAP}/usr/bin/curl" -L https://github.com/genuinetools/reg/releases/download/${REG_VERSION}/reg-linux-amd64 -o "$SNAP_DATA/bin/reg"
```

Comment:
MicroK8s ships for amd64 adn arm64, so we should use the  $SNAP_ARCH to get the right binary from the releases. Something like this should work: `https://github.com/genuinetools/reg/releases/download/${REG_VERSION}/reg-linux-${SNAP_ARCH}`.

Reply:
Alright!